stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -193,15 +208,17 @@ internal class PaymentSheetViewModel internal constructor(
         }
     }
 
-    private fun resetViewState(paymentIntent: PaymentIntent) {
+    private fun resetViewState(paymentIntent: PaymentIntent, userErrorMessage: String?) {
```

Comment:
We have this ViewState called Ready, but I wonder if I should rename it `ResetViewState`.

Reply:
Makes sense to me given the other states are also actions.

Reply:
`userErrorMessage: String? = null` so you can omit it when calling.

Reply:
I considered setting it to null as a default, but I wanted to be explicit here in that you must specify the error message - that can be null if you want, but you will be making a change to the value either way.

Reply:
Rename is happening here: https://github.com/stripe/stripe-android/pull/3674